src/lib/consensus/proof_of_stake.ml

```diff
@@ -491,48 +491,78 @@ module Vrf = struct
   module Threshold = struct
     open Bignum_bigint
 
-    let of_uint64_exn = Fn.compose of_int64_exn UInt64.to_int64
-
     (* TEMPORARY HACK FOR TESTNETS: c should be 1 otherwise *)
-    let c_int = 2
+    let c : [`One | `Two] = `Two
+
+    let c_int = match c with `One -> 1 | `Two -> 2
 
-    let c = of_int c_int
+    let base = Bignum.(one / of_int 2)
```

Comment:
Should we make this configurable? There's a tradeoff between latency-to-finality, bandwidth consumption, and (I think) fork rate. The genesis paper doesn't actually specify a value.

Reply:
+1 I definately want this to be a build configuration.